File: test_FCTrie.java
/* Names: Erin Cole and Christina Cook
 * Filename: test_FCTrie.java
 * Description: unit test for FCTrie.java
 */


class test_FCTrie {

void test_insert(String[] trieNode) {
       
     FCTrie myTrie = new FCTrie(null);
     myTrie.insert(trieNode);
}

void test_find(String[] trieNode) {
     FCTrie findTrie = new FCTrie(null);
     findTrie.find(trieNode);
}

int databaseSize = 40;
String[] trieNode = new String[databaseSize];

void main() {
     test_insert(trieNode);         
     test_find(trieNode);
}
}

File: testFCTrie.js
/* Names: Erin Cole and Christina Cook
 * Project Name: Project Fortune Cookie
 * Filename: testFCTrie.js
 * Description: This file tests the FCTrie.js functions
 */


function testFCTrie() {
	var fileKey = testText.txt;
	var myFCTrie = new FCTrie(fileKey); 

	if(myFCTrie.value)
		alert('yay it worked!');
}

function testput(obj) {
	return obj == this;
}

function testget(obj) {
	FCTrie.put(obj);
	FCTrie.get(obj);

	if (obj == FCTrie.get(obj)) {
		alert('put node into trie');
	}
}
